masonry_core/src/core/contexts.rs

```diff
@@ -861,6 +875,11 @@ impl_context_method!(
         pub fn to_window(&self, widget_point: Point) -> Point {
             self.widget_state.window_transform * widget_point
         }
+
+        /// Get the given child's size.
+        pub fn child_size(&self, child: &WidgetPod<impl Widget + ?Sized>) -> Size {
+            self.get_child_state(child).size
+        }
```

Comment:
We should document the default value before layout has been called on this method.

For example, during `WidgetAdded`, this won't be meaningfuly defined.